packages/hash/shared/src/createProseMirrorState.ts

```diff
@@ -4,28 +4,21 @@ import { keymap } from "prosemirror-keymap";
 import { ProsemirrorNode, Schema } from "prosemirror-model";
 import { EditorState, Plugin } from "prosemirror-state";
 import { createEntityStorePlugin } from "./entityStorePlugin";
-import { createSchema } from "./prosemirror";
+import { createSchema, getPageEditorNodes } from "./prosemirror";
 import { wrapEntitiesPlugin } from "./wrapEntitiesPlugin";
 
-const createInitialDoc = (schema: Schema = createSchema()) =>
-  schema.node("doc", {}, [schema.node("loading")]);
+const createInitialDoc = (
+  schema: Schema = createSchema(getPageEditorNodes()),
+) => schema.node("doc", {}, [schema.node("loading")]);
 
 const defaultPlugins: Plugin<any, Schema>[] = [
   ...wrapEntitiesPlugin(baseKeymap),
   // This enables an indicator to appear when drag and dropping blocks
   dropCursor(),
 ];
 
-export const createProseMirrorState = ({
-  accountId,
-  doc = createInitialDoc(),
-  plugins = [],
-}: {
-  accountId: string;
-  doc?: ProsemirrorNode<Schema>;
-  plugins?: Plugin<any, Schema>[];
-}) => {
-  const formatKeymap = keymap<Schema>({
+export const formatKeymap = (doc: ProsemirrorNode<Schema>) =>
```

Comment:
Since `formatKeymap` is a util function now, maybe we can move it into `packages/hash/shared/src/prosemirror.ts` ?